packages/core/core/src/RequestTracker.js

```diff
@@ -1227,6 +1231,36 @@ export default class RequestTracker {
     }
   }
 
+  flushStats() {
+    let requestTypeEntries = {};
+    type RequestTypes = Array<$Keys<typeof requestTypes>>;
+
+    for (let key of (Object.keys(requestTypes): RequestTypes)) {
+      requestTypeEntries[requestTypes[key]] = key;
+    }
+
+    let formattedStats = {};
+    let messageLines = ['Request stats'];
+
+    for (let [requestType, count] of this.stats.entries()) {
+      let requestTypeName = requestTypeEntries[requestType];
+      formattedStats[requestTypeName] = count;
+      messageLines.push(`${requestTypeName}\t\t${count}`);
+    }
+
+    let message = messageLines.join('\n');
+
+    logger.verbose({
+      origin: '@parcel/core',
+      message,
+      meta: {
+        requestStats: formattedStats,
```

Comment:
Won't formatted stats be more difficult to deal with as metrics on the other end?

I don't think `RequestTracker` should be responsible for formatting output, it should send the raw data, and the reporter that deals with it can format them for output if needed (or send the raw values to a backend).

Reply:
Might be poor variable naming, `formattedStats` is just the stats but with the requestType numeric codes translated in to the names (i.e. `{ 'asset_graph_request': 1487, ...` rather than `{ 3: 1487, ...`). It is still just raw data though

Reply:
Ah, yes, my bad - I missed the `formattedStats` vs `messageLines` being two different things used differently. All good!